src/lib/blockedLinks.js

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+/* jshint node:true, esnext:true */
+
+const async = require("lib/async");
+const gUserDatabase = require("lib/userDatabase");
+
+/**
+ * Singleton that keeps track of all blocked links in the grid.
+ */
+const gBlockedLinks = {
```

Comment:
Correct! (It's the module loader that does this actually, i.e. Webpack, not React)